Trappist/src/Promact.Trappist.Test/Category/CategoryRepositoryTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using System;
+using System.Linq;
+using Promact.Trappist.Repository.Categories;
+
+namespace Promact.Trappist.Test.Category
+{
+    [Collection("Register Dependency")]
+    public class CategoryRepositoryTest
+    {
+        private readonly Bootstrap _bootstrap;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the '_bootstrap' field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)